base64ct/Cargo.toml

```diff
@@ -14,9 +14,13 @@ categories = ["cryptography", "encoding", "no-std"]
 keywords = ["base64", "phc"]
 readme = "README.md"
 
+[dependencies]
+zeroize = ""
+
```

Comment:
Shouldn't zeroization be a concern of a higher level? It's a bit strange to have it in a parsing library.

Reply:
Removed `zeroize`